homeassistant/components/mutesync/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "m\u00fctesync",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
```

Comment:
Don't use a placeholder for invalid auth. Instruct the user to where they can open up authentication in the app.